Add info to scavenge tutorial when make install fails

**What happened:**

I ran `make install` and it failed like so:

```bash
 ~/godays2020/workshop/scaffold   master  make install
/bin/sh: go-bindata: Kommando nicht gefunden.
make: *** [Makefile:13: generate] Fehler 127
```

**Solution:**

Run `make` because this will include the step `make tools` to install `go-bindata`.

So it would be great to add this info to the tutorial in sdk-tutorials/scavenge/tutorial/03-scaffold.md

--- a/nameservice/tutorial/00-intro.md
+++ b/nameservice/tutorial/00-intro.md
@@ -60,12 +60,4 @@ Through the course of this tutorial you will create the following files that mak
         └── module.go
 ```
 
-Start by creating a new git repository:
-
-```bash
-mkdir nameservice
-cd nameservice
-git init
-```
-
-Then, just follow along! The first step describes the design of your application.
+Follow along! The first step describes the design of your application.

--- a/scavenge/tutorial/03-scaffold.md
+++ b/scavenge/tutorial/03-scaffold.md
@@ -8,6 +8,7 @@ We'll be using a tool called [scaffold](https://github.com/cosmos/scaffold) to h
 ```bash
 git clone git@github.com:cosmos/scaffold.git
 cd scaffold
+make tools
 make install
 scaffold --help
 ```

